homeassistant/components/light/flux_led.py

```diff
@@ -0,0 +1,115 @@
+"""
+Support for Flux lights.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.flux_led/
+"""
+
+import logging
+import socket
+import voluptuous as vol
+
+from homeassistant.components.light import Light
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['https://github.com/Danielhiversen/flux_led/archive/0.3.zip']
```

Comment:
The traceback I got:

```bash
16-07-17 15:49:56 INFO (MainThread) [homeassistant.loader] Loaded light.flux_led from homeassistant.components.light.flux_led
Traceback (most recent call last):
  File "/home/fab/Documents/repos/home-assistant/homeassistant/util/package.py", line 46, in check_package_exists
    req = pkg_resources.Requirement.parse(package)
  [...]
  File "/home/fab/Documents/repos/home-assistant/homeassistant/util/package.py", line 49, in check_package_exists
    req = pkg_resources.Requirement.parse(urlparse(package).fragment)
  File "/home/fab/Documents/repos/home-assistant/lib64/python3.4/site-packages/pkg_resources/__init__.py", line 3047, in parse
    req, = parse_requirements(s)
ValueError: need more than 0 values to unpack
```
I guess that it could be fixed by changing the `REQUIREMENTS` (untested) -> `REQUIREMENTS = ['https://github.com/Danielhiversen/flux_led/archive/0.3.zip#flux_led==0.3']`